src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -442,9 +442,13 @@ const SendAssetsForm = ({ txConfig, onComplete, protectTxButton, isDemoMode }: P
     handleNonceEstimate(values.account);
   }, [values.account]);
 
-  useEffect(() => {
-    handleGasEstimate();
-  }, [values.account, values.address]);
+  useDebounce(
```

Comment:
Can we migrate to saga instead?

Reply:
Refactoring the Send flow is not the priority at the moment. Getting polished is. So I would say we wait with that until we refactor it further.